percona-server-8.3.yaml

```
@@ -0,0 +1,147 @@
+      expected-commit: fcee26ff5d51b9e72fa8b3b626a3424e2cbe2794
+
+  - name: "Update Submodules"
+    runs: |
+      git submodule update --init --recursive
+
+  - name: "Remove Coredumper Support"
+    runs: |
+      sed -i 's/INCLUDE(coredumper)//g' CMakeLists.txt
+      sed -i 's/MYSQL_CHECK_COREDUMPER()//g' CMakeLists.txt
+      sed -i 's/OPTION (WITH_COREDUMPER "Enable support for coredumper library" ON)//g' CMakeLists.txt
+
+  - name: "Licencing"
+    runs: |
+      mkdir /usr/share/percona-server
+      mkdir /usr/share/percona-server/licenses
+      mkdir /usr/share/percona-server/licenses/curl
+      mkdir /usr/share/percona-server/licenses/zstd
+      mkdir /usr/share/percona-server/licenses/innobase
+      mkdir /usr/share/percona-server/licenses/libtirpc
+      mkdir /usr/share/percona-server/licenses/innodb
+      mkdir /usr/share/percona-server/licenses/libedit
+      mkdir /usr/share/percona-server/licenses/cmake
+
+      cp extra/curl/curl-8.4.0/COPYING ../../usr/share/percona-server/licenses/curl/
+      cp extra/zstd/zstd-1.5.5/COPYING ../../usr/share/percona-server/licenses/zstd/
+      cp storage/innobase/COPYING.Google ../../usr/share/percona-server/licenses/innobase/
+      cp storage/innobase/COPYING.Percona ../../usr/share/percona-server/licenses/innobase/
+      cp extra/tirpc/libtirpc-1.3.3/COPYING ../../usr/share/percona-server/licenses/libtirpc/
+      cp COPYING.innodb-deadlock-count-patch ../../usr/share/percona-server/licenses/innodb/
+      cp extra/libedit/libedit-20221030-3.1/COPYING ../../usr/share/percona-server/licenses/libedit/
+      cp extra/libevent/libevent-2.1.11-stable/cmake/COPYING-CMAKE-SCRIPTS ../../usr/share/percona-server/licenses/cmake/
+
+  - name: "Cmake"
+    runs: |
+      mkdir /var/tmp
+      cmake . -G Ninja \
+        -DFORCE_INSOURCE_BUILD=1 \
+        -DBUILD_CONFIG=mysql_release \
+        -DWITH_EDITLINE=system \
+        -DCMAKE_INSTALL_PREFIX=/usr \
+        -DSYSCONFDIR=/etc \
+        -DMYSQL_DATADIR=/var/lib/mysql \
+        -DMYSQL_UNIX_ADDR=/run/mysqld/mysqld.sock \
+        -DDEFAULT_CHARSET=utf8mb4 \
+        -DDEFAULT_COLLATION=utf8mb4_general_ci \
+        -DENABLED_LOCAL_INFILE=ON \
+        -DINSTALL_INFODIR=share/info \
+        -DINSTALL_MANDIR=share/man \
+        -DINSTALL_PLUGINDIR=lib/${{package.name}}/plugin \
+        -DINSTALL_INCLUDEDIR=include/mysql \
+        -DINSTALL_DOCREADMEDIR=share/doc/${{package.name}} \
+        -DINSTALL_SUPPORTFILESDIR=share/${{package.name}} \
+        -DINSTALL_DOCDIR=share/doc/${{package.name}} \
+        -DWITH_ROCKSDB=NO \
+        -DTMPDIR=/var/tmp \
+        -DWITH_ASAN=OFF \
+        -DWITH_EXTRA_CHARSETS=complex \
+        -DWITH_SYSTEMD=no \
+        -DWITH_SSL=system \
+        -DWITH_VALGRIND=OFF \
+        -DWITH_ZLIB=system \
+
+  - name: "Install"
+    runs: |
+      DESTDIR="${{targets.destdir}}" ninja install
+
+  - name: "Remove extras"
+    runs: |
+      rm -rf "${{targets.destdir}}"/usr/local/mysql/mysql-test/
```

```suggestion
      rm -rf "${{targets.destdir}}"/usr/mysql-test
```

I think this is the correct path to remove since the install prefix is set to /usr